infra/main.bicep

```diff
@@ -1129,6 +1129,17 @@ module storageRoleContributorSearchService 'core/security/role.bicep' = if (useI
   }
 }
 
+// Necessary for the Container Apps backend to store tokens in the container
+module storageRoleContributorBackend 'core/security/role.bicep' = if (deploymentTarget == 'containerapps' && !empty(clientAppId)) {
```

Comment:
Note: If the user upload feature is also enabled, then the ACA backend will get granted both "Storage Blob Data Owner" and "Storage Blob Data Contributor". That should work fine, even though it's a bit redundant.